systers_portal/static/css/style.css

```diff
@@ -234,6 +234,22 @@ table.decoration-none  tr a {
   font-size: 1.3em;
 }
 
+/* Meetup Location CSS
+------------------------------------------------- */
+.box-boundary{
+  outline: 1px solid black;
+}
+
+.box-container{
+  margin: 10px 40px 10px;
+}
+
+.box-body{
+  font-size: 16px;
```

Comment:
Please use relative units, like **em** for font-size/line-height.
In general I would suggest applying same styles as `.blog-entry .body` selector. You can do that adding your custom box selector separated by a comma.